bin/merge.sh

```diff
@@ -19,18 +19,15 @@
 # limitations under the License.
```

Comment:
Question:
Which parts of the "Files changed" are from `cv32e40s/dev` and which are from the manual conflict resolution?

Answer:
Diff of conflict resolution https://github.com/silabs-robin/core-v-verif/compare/release__merge_conflicts_from__dev..openhwgroup:core-v-verif:cv32e40s/dev-1

Reply:
@MikeOpenHWGroup, this was the only manual changes you did, right?

Everything else in this PR's diff are changes that come from the `/dev` branch?

Reply:
And resolving the merge conflicts involves only the diff shown in the above link?

Reply:
Yes, that is correct.